spec/api-browser-window-spec.js

```diff
@@ -1470,29 +1472,6 @@ describe('BrowserWindow module', () => {
 
       const preload = path.join(fixtures, 'module', 'preload-sandbox.js')
 
-      // http protocol to simulate accessing another domain. This is required
```

Comment:
@ppontes is there any reason why you removed the usage of `interceptStringProtocol`? I'm considering using this again for https://github.com/electron/electron/pull/18173

Reply:
Unfortunately, I don't remember the reason. Feel free to try to use it again according to your needs.